news/feature-3911.md

```diff
@@ -0,0 +1,15 @@
+`$(values)` and `$(names)`: these new template functions can be used to
+query a list of name-value pairs in the current message. The list of name
+value pairs queried are specified by a value-pairs expression, just like
+with `$(format-json)`.
+
+Examples:
+
+  This expression sets the JSON array `values` to contain the list of SDATA
+  values, while the JSON array `names` would contain the associated names, in
+  the same order.
+
+  $(format-json values=list($values .SDATA.*) names=list($(names .$SDATA.*)))
```

Comment:
The example config is incorrect: `list($values .SDATA.*)`, the correct `list($(values .SDATA.*))` (missing parenthesis for value fn.
The names invocation is also not desired example (but not incorret), should be `list($(names .SDATA.*))` (the `$` is not needed).

Reply:
I've fixed the example.
@Kokan Please check it.